### PR TITLE
Added more channels

### PR DESCRIPTION
**Pull request by the-ethan-hunt**
solved #10 . Added description for all ACTIVE channels.

### CODE REVIEW

File: channel_long_descriptions.json [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
+  "cute-animal-pics": "A place to watch photos of animals cuddling or playing together with the usual gossip!",
+  "elections": "Witness history being created when students contest with each other for the most coveted posts in KGP",
   "flashpoint": "An alternate timeline for Facebook posts",
-  "random": "The channel for everything that doesn't fit anywhere above!"
+  "general": "This is the channel for all general discussion. This could be related to any topic of your choice, a concern or otherwise. Feel free to tag any single person or a group of people to start a discussion. Remember that mentioning the whole channel using is strictly prohibited",
+  "github": "Get notifications from GitHub about recent PRs to Meta projects",
+  "gsoc": "A channel for discussing recent updates or any other stuff related to the prestigious Google Summer of Code",
+  "meetup": "A channel for discussing upcoming MetaKGP meet-ups",
+  "meta-x": "A channel for discussing quirky projects, useful hacks and a lot more that pushes the boundaries of how we live and interact in KGP",
+  "music": "A chaneel where you can share your musical tastes and the latest videos of your favourite singers!",
```
Comment (athityakumar) [inline channel_long_descriptions.json]: *channel

File: channel_long_descriptions.json [hunk 2]
```diff
@@ -1,6 +1,16 @@
 {
 
-  "general": "This is the channel for all general discussion. This could be related to any topic of your choice, a concern or otherwise. Feel free to tag any single person or a group of people to start a discussion. Remember that mentioning the whole channel using is strictly prohibited",
+  "book-club": "The place where bibliophiles and MetaKGPians meet to discuss (spoilers!)",
```
Comment (athityakumar) [inline channel_long_descriptions.json]: The place where bibliophiles meet to discuss about books, and also occasionally rant about "Never judge a book by it's TV series /  movie".

File: channel_long_descriptions.json [hunk 3]
```diff
@@ -1,6 +1,16 @@
...
+  "cute-animal-pics": "A place to watch photos of animals cuddling or playing together with the usual gossip!",
```
Comment (athityakumar) [inline channel_long_descriptions.json]: No gossip here, really! Just cute animal pictures! :wink:

File: channel_long_descriptions.json [hunk 4]
```diff
@@ -1,6 +1,16 @@
...
+  "elections": "Witness history being created when students contest with each other for the most coveted posts in KGP",
```
Comment (athityakumar) [inline channel_long_descriptions.json]: "Most coveted posts in KGP" -> "post of maintainer(s) of Metakgp".

File: channel_long_descriptions.json [hunk 5]
```diff
@@ -1,6 +1,16 @@
...
-  "random": "The channel for everything that doesn't fit anywhere above!"
+  "general": "This is the channel for all general discussion. This could be related to any topic of your choice, a concern or otherwise. Feel free to tag any single person or a group of people to start a discussion. Remember that mentioning the whole channel using is strictly prohibited",
+  "github": "Get notifications from GitHub about recent PRs to Meta projects",
+  "gsoc": "A channel for discussing recent updates or any other stuff related to the prestigious Google Summer of Code",
+  "meetup": "A channel for discussing upcoming MetaKGP meet-ups",
+  "meta-x": "A channel for discussing quirky projects, useful hacks and a lot more that pushes the boundaries of how we live and interact in KGP",
+  "music": "A chaneel where you can share your musical tastes and the latest videos of your favourite singers!",
+  "random": "The channel for everything that doesn't fit anywhere above!",
+  "recent-changes": "Changes related to KGP brought to you by our very own Batman!",
+  "tech-help": "All queries to Linux,Ubuntu and Meta-projects are discussed here. Please research well,before asking your questions"
```
Comment (athityakumar) [inline channel_long_descriptions.json]: research or search? Also, maybe we can point to a few sources like Google / Stack Overflow.

Also add proper spacing for commas. That is, no space before comma, and one space after a comma - like this sentence.

**Comment by the-ethan-hunt** [comment]
changes as suggested by @athityakumar added!

**Comment by athityakumar** [comment]
Looks good to go. Merging this in. :+1:

**Pull request merged**